src/containers/Attributes/CreateAttribute.tsx

```diff
@@ -30,6 +30,7 @@ const CreateAttribute: FC<Props> = (props) => {
   const handleCreateAuthority = useCallback(
     async ({ authority }) => {
       try {
+        authority = authority.replace(/\/+$/g, '');
```

Comment:
[Sonarcloud doesn't like using a regex here](https://sonarcloud.io/project/security_hotspots?id=opentdf_frontend&pullRequest=166&resolved=false&types=SECURITY_HOTSPOT), so why not be more clear and use

```suggestion
        if (authority.endsWith('/')) {
          authority = authority.slice(0, -1);
        }
```

Reply:
updated!